packages/container/src/container.tsx

```diff
@@ -12,7 +12,7 @@ export const Container = React.forwardRef<
   ContainerElement,
   Readonly<ContainerProps>
 >(({ children, style, ...props }, forwardedRef) => {
-  const styles = { maxWidth: '37.5em', ...style };
+  const styles: React.CSSProperties = { maxWidth: '37.5em', ...style };
```

Comment:
I think there's not need to add the types here, since it's inferred.

Reply:
oh，you are right.